src/components/AppNavigation/AppNavigationHeader/AppNavigationHeaderNewEvent.vue

```diff
@@ -22,14 +22,16 @@
 
 <template>
 	<NcButton v-shortkey="['c']"
+		:aria-label="newEvent"
 		class="button new-event"
+		:title="newEvent"
```

Comment:
**issue:** Some screen-readers read **both** `aria-label` and `title`, so you don't want to put both. I think having only `aria-label` is more appropriate.

Reply:
@tcitworld check my 3rd commit :-)